core/src/main/java/com/klaviyo/coresdk/KlaviyoConfig.kt

```diff
@@ -12,7 +11,8 @@ object KlaviyoConfig {
     private const val NETWORK_TIMEOUT_DEFAULT: Int = 500
     private const val NETWORK_FLUSH_INTERVAL_DEFAULT: Int = 60000
     private const val NETWORK_FLUSH_DEPTH_DEFAULT: Int = 20
-    private const val NETWORK_FLUSH_CHECK_INTERVAL: Long = 2000
+    private const val NETWORK_FLUSH_CHECK_INTERVAL: Int = 2000
+    private const val NETWORK_USE_ANALYTICS_BATCH_QUEUE: Boolean = true
```

Comment:
Adding activation of the batch queue to the configuration. Will be useful for early day testing